package.json

```diff
@@ -33,7 +33,8 @@
     "ioredis-mock": "^5.5.5",
     "jsonwebtoken": "^8.5.1",
     "pino": "^6.11.3",
-    "pino-colada": "^2.1.0"
+    "pino-colada": "^2.1.0",
+    "node-fetch": "^2.6.1"
```

Comment:
Instead of manually adding dependencies to `package.json`, do it via `npm install --save` or `--save-dev` so that they get inserted in alphabetical order.